services/comments/comments.go

```diff
@@ -25,10 +25,6 @@ func CreateIssueComment(doer *models.User, repo *models.Repository, issue *model
 	if err != nil {
 		return nil, err
 	}
-	err = issues.SaveIssueContentHistory(db.GetEngine(db.DefaultContext), doer.ID, issue.ID, comment.ID, timeutil.TimeStampNow(), comment.Content, true)
```

Comment:
Why remove these?

Reply:
Discussed above. This PR introduces a new method to handle the history:

1. A newly created issue/comment do not record the history.
2. When the content is edited, generate the first_created history record and edited history record.